tasks/broad/CopyFilesFromCloudToCloud.wdl

```diff
@@ -22,7 +22,7 @@ task CopyFilesFromCloudToCloud {
     String vault_token_path
     String google_account_vault_path
     Float? contamination
-    String base_file_name
+    String base_file_name = "base_file"
```

Comment:
did you mean to hard code "base_file" here? maybe there is some kind of input id to pass here

Reply:
I did actualy, some pipelines will produce outputs that are non-file i.e. for WGS it produces a contamination score which is a float. When this is done you need to copy that float to a file and name it with `base_file_name`...i added a fall back name (which will be unused) for pipelines that don't produce nonfile output